shared/src/shapes/PlayerReminder.ts

```diff
@@ -3,7 +3,7 @@ import { z } from "zod";
 export const playerReminderShape = z.object({
   name: z.string(),
   toPlayer: z.string(),
-  fromPlayer: z.string().optional(),
+  fromPlayer: z.string(),
```

Comment:
This should be optional later for e.g. Delusional roles, but until that's implemented I'm happy to keep this mandatory for now.